### ISSUE TITLE
Is it posible to use it for a event the same day

### ISSUE THREAD

**Issue by xoco70** [krystalcampioni/vue-hotel-datepicker#21]
Hi, 

I would like to use very much you plugin, because I think it is very nice.

My use case is for tournaments, and it can happen that the event is 1 or more day.

Is it possible to select the same date for the check-in and the check-out??

**Comment by lucasjahn** [comment]
Hey there,

this is totally possible, you just have to set minNights to 0, then it should work like you want.
greetz ;)

**Comment by xoco70** [comment]
mmm. It is not working.

Can you please confirm this is what I need?

```
                    <HotelDatePicker
                            format="DD-MM-YYYY"
                            minNights="0"
                    ></HotelDatePicker>
``` 
When I do that, after first date selection, dates are selected till March 8th. 

I aslo tried `minNights=0` with no commas, but still not working

**Comment by xoco70** [comment]
I get : 

`[Vue warn]: Invalid prop: type check failed for prop "minNights". Expected Number, got String.` even if I put it with no commas

**Comment by xoco70** [comment]
Ok, I have no more error with :
```
<HotelDatePicker
                            format="DD-MM-YYYY"
                            :minNights=minNights ></HotelDatePicker>
```

and with 

```
data() {
            return {
                 minNights: 0,
            }
```


But I still can't select the same day for the end. I get a forbidden cursor on hover

**Comment by lucasjahn** [comment]
Hmm interesting, I can't check it now but it should be in place as of this fixed pull request #14. Maybe you have to redownload and update your package.json with the newest version if this module. Maybe you just have an older version without this fix in it.

**Comment by xoco70** [comment]
I deleted it, and installed it again, but it didn't work.

It seems it downloads: "vue-hotel-datepicker@2.0.6",

it is the good version?

**Comment by krystalcampioni** [comment]
@xoco70 if you want to pass a value as a number (or an array, object, ect)  instead of a string to a component via prop in Vue, you have to use `v-bind:` or `:`. This means you can use `:minNights="0"` and don't necessarily have to save this zero in a data property. These options allow the following behavior: 
1. Click on date to select the check-in
2. Click on the same date again to select the check-out

Is this the behavior you were expecting?

**Comment by jeffreybrayne** [comment]
I am having the same issue. I've set `:minNights="0"` and am still unable to select the same day for end date.

**Comment by krystalcampioni** [comment]
@jeffreybrayne @xoco70 I found out that the previous versions only allowed you to choose the same day for check-in and check-out not only if the the `minNights` was set to `0` but also if the `enableCheckout` was `true`. Just fixed that on v2.0.9, please update the version on your project.
